phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ApplicationSideBar.java

```diff
@@ -158,7 +158,7 @@ private ToggleButton createAllCategoriesToggleButton() {
     private ToggleButton createCategoryToggleButton(CategoryDTO category) {
         final LeftToggleButton categoryButton = new LeftToggleButton(category.getName());
 
-        categoryButton.setId(String.format("%sButton", category.getName().toLowerCase()));
+        categoryButton.setId(String.format("%sButton", category.getId().toLowerCase()));
```

Comment:
`name` is the displayed name and might therefore be translated whereas `id` is a unique identifier (currently simply the English name).